src/index.js

```diff
@@ -406,7 +420,6 @@ loginModalSelector.addEventListener('click', (event) => {
 })
 
 
-
 loginSubmitButton.addEventListener('click', (event) => {
   event.preventDefault()
 
```

Comment:
why are you putting the getUserData fetch here instead of just calling it? It would make it a bit more clear when reading through your code.

Reply:
I tried writing a function to call it, but it wouldn't work. It's basically just collected when someone logs in as a customer, otherwise, I don't need it. I'll look closer at refactoring that to make more sense.